### ISSUE TITLE
Backward/Forward word delete does not respect word separators

### ISSUE THREAD

**Issue by m27315** [tuttieee/vscode-emacs-mcx#106]
# Steps to Reproduce

1. Modify `settings.json` to add new separators (e.g., underscore):
```
    "editor.wordSeparators": "`~!@#$%^&*()-=+[{]}\\|;:'\",.<>/?_"
```
2. Observe that move-forward-word and move-backward-word honor underscore as word boundary.
3. Press M-d (backward-kill-word) or M-backspace (backward-kill-word) and note that it does not respect underscore as a word separator.

# Versions
```
Version: 1.33.1
Awesome Emacs Keymap: 0.11.2
Commit: 51b0b28134d51361cf996d2f0a1c698247aeabd8
Date: 2019-04-11T08:20:22.771Z
Electron: 3.1.6
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Linux x64 4.19.27-gentoo-r1
```

**Comment by joeshaw** [comment]
I've found that it doesn't obey existing separators like `"` as well, but only in some language modes.  For instance, if I type `"foo"` with my cursor at the second `o` and hit M-backspace, in Go it does the right thing (leaving `""`) but in JSON it leaves `"`.

I'm not sure if that's something specific to this extension or the language mode, though.

**Comment by m27315** [comment]
I'm glad to know that it's not just me. ... Has anything changed in the past several months, as far as you know, to enable resolution? ... Thanks! :smile:

**Comment by tuttieee** [comment]
#321 

Fixed in 0.20.4 👍
